environments/sandbox.yml

```diff
@@ -36,7 +36,7 @@ A:
   - name: "portal"
     ttl: 60
     record:
-      - "20.49.172.208"
+      - "51.132.51.56"
```

Comment:
if this will be changing often you can probably just create the entry as part of the automation?